ui/src/main/resources/FileManager/FileManagerTranslations.pt_BR.xml

```diff
@@ -0,0 +1,90 @@
+<xwikidoc>
+  <web>FileManager</web>
+  <name>FileManagerTranslations</name>
+  <language/>
+  <defaultLanguage/>
```

Comment:
The language and default language fields are not right. The language should be pt_BR and the default language should be en. I'm not sure how you generated this document.